setup.py

```diff
@@ -5,8 +5,8 @@
     "pytest-xdist>=2",
     "coverage>=5",
     "pytest-cov>=2.10.1",
+    "networkx~=2.4",
```

Comment:
why is it still a dev dependency?

Reply:
It's used in the tests at least for now.